### ISSUE TITLE
Failed build on Freebsd 10.2

### ISSUE THREAD

**Issue by Martinfx** [kcat/openal-soft#33]
Hi i build on freebsd 10.2-RELEASE-p11 
FreeBSD clang version 3.4.1 (tags/RELEASE_34/dot1-final 208032) 20140512

```
cd build
cmake .. 
make -j8
```
but compiler says error:
```

Scanning dependencies of target bsincgen
Scanning dependencies of target makehrtf
Scanning dependencies of target test-common
Scanning dependencies of target common
[  1%] Building C object CMakeFiles/bsincgen.dir/utils/bsincgen.c.o
[  4%] Building C object CMakeFiles/test-common.dir/examples/common/alhelpers.c.o
[  4%] Building C object CMakeFiles/makehrtf.dir/utils/makehrtf.c.o
[  6%] Building C object CMakeFiles/common.dir/common/atomic.c.o
[  8%] Building C object CMakeFiles/common.dir/common/threads.c.o
[ 11%] Building C object CMakeFiles/common.dir/common/rwlock.c.o
[ 11%] Building C object CMakeFiles/common.dir/common/uintmap.c.o
[ 13%] Linking C static library libtest-common.a
[ 14%] Linking C executable bsincgen
[ 16%] Linking C static library libcommon.a
[ 16%] Built target test-common
[ 16%] Built target common
[ 16%] Built target bsincgen
Scanning dependencies of target openal
[ 18%] Building C object CMakeFiles/openal.dir/OpenAL32/alAuxEffectSlot.c.o
[ 21%] Building C object CMakeFiles/openal.dir/OpenAL32/alEffect.c.o
[ 19%] Building C object CMakeFiles/openal.dir/OpenAL32/alError.c.o
[ 22%] Building C object CMakeFiles/openal.dir/OpenAL32/alBuffer.c.o
[ 24%] Building C object CMakeFiles/openal.dir/OpenAL32/alFilter.c.o
[ 26%] Building C object CMakeFiles/openal.dir/OpenAL32/alExtension.c.o
[ 27%] Building C object CMakeFiles/openal.dir/OpenAL32/alListener.c.o
[ 29%] Building C object CMakeFiles/openal.dir/OpenAL32/alSource.c.o
[ 31%] Building C object CMakeFiles/openal.dir/OpenAL32/alState.c.o
[ 32%] Building C object CMakeFiles/openal.dir/OpenAL32/alThunk.c.o
[ 34%] Building C object CMakeFiles/openal.dir/OpenAL32/sample_cvt.c.o
[ 36%] Building C object CMakeFiles/openal.dir/Alc/ALc.c.o
[ 37%] Building C object CMakeFiles/openal.dir/Alc/ALu.c.o
[ 39%] Building C object CMakeFiles/openal.dir/Alc/alcConfig.c.o
[ 40%] Building C object CMakeFiles/openal.dir/Alc/alcRing.c.o
[ 42%] Building C object CMakeFiles/openal.dir/Alc/bs2b.c.o
[ 44%] Building C object CMakeFiles/openal.dir/Alc/effects/autowah.c.o
[ 45%] Building C object CMakeFiles/openal.dir/Alc/effects/chorus.c.o
[ 47%] Building C object CMakeFiles/openal.dir/Alc/effects/compressor.c.o
[ 49%] Building C object CMakeFiles/openal.dir/Alc/effects/dedicated.c.o
[ 50%] Building C object CMakeFiles/openal.dir/Alc/effects/distortion.c.o
[ 52%] Building C object CMakeFiles/openal.dir/Alc/effects/echo.c.o
[ 54%] Building C object CMakeFiles/openal.dir/Alc/effects/equalizer.c.o
[ 55%] Building C object CMakeFiles/openal.dir/Alc/effects/flanger.c.o
[ 57%] Linking C executable makehrtf
[ 57%] Built target makehrtf
[ 59%] Building C object CMakeFiles/openal.dir/Alc/effects/modulator.c.o
[ 60%] Building C object CMakeFiles/openal.dir/Alc/effects/null.c.o
[ 62%] Building C object CMakeFiles/openal.dir/Alc/effects/reverb.c.o
[ 63%] Building C object CMakeFiles/openal.dir/Alc/helpers.c.o
[ 65%] Building C object CMakeFiles/openal.dir/Alc/bsinc.c.o
[ 67%] Building C object CMakeFiles/openal.dir/Alc/panning.c.o
[ 68%] Building C object CMakeFiles/openal.dir/Alc/hrtf.c.o
[ 70%] Building C object CMakeFiles/openal.dir/Alc/mixer.c.o
[ 72%] Building C object CMakeFiles/openal.dir/Alc/mixer_c.c.o
[ 73%] Building C object CMakeFiles/openal.dir/Alc/mixer_sse.c.o
[ 75%] Building C object CMakeFiles/openal.dir/Alc/mixer_sse2.c.o
[ 77%] Building C object CMakeFiles/openal.dir/Alc/mixer_sse3.c.o
[ 78%] Building C object CMakeFiles/openal.dir/Alc/mixer_sse41.c.o
[ 80%] Building C object CMakeFiles/openal.dir/Alc/backends/base.c.o
[ 81%] Building C object CMakeFiles/openal.dir/Alc/backends/loopback.c.o
[ 83%] Building C object CMakeFiles/openal.dir/Alc/backends/null.c.o
[ 85%] Building C object CMakeFiles/openal.dir/Alc/backends/alsa.c.o
[ 86%] Building C object CMakeFiles/openal.dir/Alc/backends/oss.c.o
[ 88%] Building C object CMakeFiles/openal.dir/Alc/backends/portaudio.c.o
[ 90%] Building C object CMakeFiles/openal.dir/Alc/backends/pulseaudio.c.o
[ 91%] Building C object CMakeFiles/openal.dir/Alc/backends/wave.c.o
/usr/home/razor/Documents/openal-soft/Alc/backends/alsa.c:592:18: error: duplicate case value '-32'
            case -EPIPE:
                 ^
/usr/home/razor/Documents/openal-soft/Alc/backends/alsa.c:591:18: note: previous case defined here
            case -ESTRPIPE:
                 ^
1 error generated.
--- CMakeFiles/openal.dir/Alc/backends/alsa.c.o ---
*** [CMakeFiles/openal.dir/Alc/backends/alsa.c.o] Error code 1

make[2]: stopped in /usr/home/razor/Documents/openal-soft/build
1 error

make[2]: stopped in /usr/home/razor/Documents/openal-soft/build
--- CMakeFiles/openal.dir/all ---
*** [CMakeFiles/openal.dir/all] Error code 2

make[1]: stopped in /usr/home/razor/Documents/openal-soft/build
1 error

make[1]: stopped in /usr/home/razor/Documents/openal-soft/build
*** [all] Error code 2

make: stopped in /usr/home/razor/Documents/openal-soft/build
1 error

make: stopped in /usr/home/razor/Documents/openal-soft/build



[build.zip](https://github.com/kcat/openal-soft/files/140814/build.zip)


```

**Comment by kcat** [comment]
Should be fixed now in commit 2c6e59c4ca50f186d19e1fd95fe058ea352c8900.

**Comment by Martinfx** [comment]
Thanks for fix.
